### ISSUE TITLE
FP: public property should be protected because already defined in parent

### ISSUE THREAD

**Issue by CZechBoY** [kalessil/phpinspectionsea#569]
Hi,
I have these classes
```php
class Base
{
    public $abc;
}
class Child extends Base
{
    public $abc;
}
``` 
which gives me this error:
```
Likely needs to be protected (already defined in Base).
```

Protected < public so the warning here is useless.

**Comment by kalessil** [comment]
Ooops, I'll skip public fields inspection.

**Comment by kalessil** [comment]
Can not reproduce: message differs.

![image](https://user-images.githubusercontent.com/1577185/31276501-751bf5ee-aa9c-11e7-936b-85fc1c1b1fe6.png)

@CZechBoY: can you please re-check your case, please?

**Comment by CZechBoY** [comment]
Yep, it is not bug. It correctly reports this issue.

**Comment by kalessil** [comment]
Uh, thanks for checking =)
